### PR TITLE
Update/combine shikigami location lists

### PR DESCRIPTION
**Pull request by Zynro**
What can I say except 
Good luck

### CODE REVIEW

File: cogs/onmyoji.py [hunk 1]
```diff
@@ -22,9 +24,17 @@
...
+number_dict = {'1':'First', '2':'Second', '3':'Third', '4':'Fourth', '5':'Fifth', '6':'Sixth', '7':'Seventh',1:'First', 2:'Second', 3:'Third', 4:'Fourth', 5:'Fifth', 6:'Sixth', 7:'Seventh'}
+
+def bold(text):
     '''Returns the Discord bolded version of input text.'''
-    return f'**{input}**'
+    return f'**{text}**'
+
+def lower_and_underscore(text):
+    """Lowers entire string and replaces all whitespace with underscores."""
+    text = text.lower()
```
Comment (wmak) [inline cogs/onmyoji.py]:
```suggestion
    return text.lower().replace(' ', '_')
```

File: cogs/onmyoji.py [hunk 2]
```diff
@@ -36,30 +46,92 @@ class DriveAPI:
...
+            sub_locale = []
+            location = row[0]
+            for again in main_sub_and_shiki_list:
+                if location == again[0]:
+                    sub_locale.append(again[1])
+            for row in main_sub_and_shiki_list:
+                if row[0] == location:
+                    main_sub_and_shiki_list.remove(row)
+            appending = [location, sub_locale]
+            final_result.append(appending)
```
Comment (wmak) [inline cogs/onmyoji.py]:
```suggestion
            final_result.append([location, sub_locale])
```

File: cogs/onmyoji.py [hunk 3]
```diff
@@ -36,30 +46,92 @@ class DriveAPI:
...
+        return(final_result)
```
Comment (wmak) [inline cogs/onmyoji.py]: are the brackets here on purpose?

File: cogs/onmyoji.py [hunk 4]
```diff
@@ -153,18 +269,100 @@ def location_finder(self, shiki):
...
+                temp_list.append(row[1])
+                for each in contains:
+                    if name.lower() in each.lower():
+                        temp_list.append(each)
+                        break
+                locale_list.append(temp_list)
+        if len(locale_list)==0:
+            print('Entry not found in DB.')
+            return
+        main_sub_and_shiki_list = []
```
Comment (wmak) [inline cogs/onmyoji.py]: This section looks identical to `generate_bbt_locations` should probably reuse the function.

File: cogs/onmyoji.py [hunk 5]
```diff
@@ -153,18 +269,100 @@ def location_finder(self, shiki):
...
                 return
             if search in self.shikigami_class[shiki].alias.lower():
                 await ctx.send(self.shiki_found(shiki))
                 await ctx.send(self.location_finder(shiki))
                 return
         await ctx.send("For all my bath powers, I could not find your term, or something went wrong.")
+
+    @commands.command()
+    async def tengu(self, ctx):
+        """hurrhurrhurr"""
```
Comment (wmak) [inline cogs/onmyoji.py]: lol?

**Review by wmak — APPROVED** [review]
hmmm I feel like there might be a better way to do images but w/e

Some shit you should fix first imo but looks good enough to me

File: cogs/onmyoji.py (hunk 5 above)
Comment (Zynro) [reply cogs/onmyoji.py]: hurrhurrhurr

File: cogs/onmyoji.py (hunk 4 above)
Comment (Zynro) [reply cogs/onmyoji.py]: oh this was dummy code to test the function

File: cogs/onmyoji.py (hunk 3 above)
Comment (Zynro) [reply cogs/onmyoji.py]: no i goofed

**Pull request merged**